code/modules/mob/say.dm

```diff
@@ -6,19 +6,46 @@
 	set category = "IC"
 	return
 
+
+/mob/verb/say_wrapper()
+	set name = "Say verb"
+	set category = "IC"
+
+	set_typing_indicator(1)
+	hud_typing = 1
```

Comment:
Could you please use TRUE/FALSE as boolean values? Applies to the rest of PR as well.